package.json

```diff
@@ -47,7 +47,7 @@
     "fast-json-patch": "github:wet-boew/JSON-Patch#wb1.0+1.1.4",
     "html5shiv": "^3.7.3",
     "jquery": "2.2.4",
-    "jquery-validation": "1.19.3",
+    "jquery-validation": "1.19.4",
```

Comment:
This change should not be here in this diff. It was already applied in a previous commit in our repository. That means you have a conflict into your commit. Have you tried to `rebase upstream/master`

If you are not able to fix this conflict issue, then the easiest is to start fresh (`reset --hard upstream/master`) and redo your change (copy/paste) by using this diff or your fork.